integration-tests/metrics-expected-changes.patch

```diff
@@ -6575,6 +6685,43 @@
    public Response response5xxMetered() {
      return Response.status(Response.Status.INTERNAL_SERVER_ERROR).build();
    }
+--- a/metrics-jersey2/src/test/java/io/dropwizard/metrics5/jersey2/resources/InstrumentedResourceExceptionMeteredPerClass.java
++++ b/metrics-jersey2/src/test/java/io/dropwizard/metrics5/jersey2/resources/InstrumentedResourceExceptionMeteredPerClass.java
+@@ -16,7 +16,7 @@ public class InstrumentedResourceExceptionMeteredPerClass {
+ 
+   @GET
+   @Path("/exception-metered")
+-  public String exceptionMetered(@QueryParam("splode") @DefaultValue("false") boolean splode)
++  public String exceptionMetered(@DefaultValue("false") @QueryParam("splode") boolean splode)
```

Comment:
One can argue that this isn't a very good change. Not a deal-breaker, but let's keep an eye on feedback.

Reply:
Yeah let's see in the internal PRs 😄